flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -763,7 +783,7 @@ CatalogBaseTable instantiateCatalogTable(Table hiveTable) {
             tableSchemaProps.putProperties(properties);
             // try to get table schema with both new and old (1.10) key, in order to support tables
             // created in old version
-            tableSchema =
+            TableSchema tableSchema =
```

Comment:
I think its doesn't matter. Since we still use the `DescriptorProperties` to se/de the schema to store/restore to the external meta store. So `TableSchema` actually still used in the hive connector system. It can be entirely removed after we can use the new way to se/de the schema. But I think we can improve it as a follow up. WDYT ?

Reply:
`CatalogPropertiesUtil` is the alternative of `DescriptorProperties`. It should not be a major work to migrate.

Reply:
Updated. Now `import org.apache.flink.table.api.TableSchema` has entirely removed from the hive connector. And the `DescriptorProperties` is migrate to `CatalogPropertiesUtil`